src/send/SendSelectRecipient.tsx

```diff
@@ -40,8 +41,14 @@ function GetStartedSection() {
   }) => {
     return (
       <View key={`getStartedOption-${optionNum}`} style={getStartedStyles.optionWrapper}>
-        <CircledIcon radius={24} style={getStartedStyles.optionNum} backgroundColor={colors.white}>
-          <Text style={getStartedStyles.optionNumText}>{optionNum}</Text>
+        <CircledIcon
+          radius={Math.min(24 * getFontScaleSync(), 50)}
+          style={getStartedStyles.optionNum}
+          backgroundColor={colors.white}
+        >
+          <Text adjustsFontSizeToFit={true} style={getStartedStyles.optionNumText}>
+            {optionNum}
+          </Text>
```

Comment:
I am seeing this number disappear on very large font sizes
<img width="250" src="https://github.com/valora-inc/wallet/assets/5062591/337f6fd2-6000-4276-8917-19c2060186de" />

Reply:
This is a bit tricky, if you change your font size while on the screen the getFontScaleSync() will not refresh; however, if you go back and navigate to the screen then it should display correctly. This is a bit of an edge case where a user adjusts their font size while this screen displayed.